src/parse.rs

```diff
@@ -172,6 +175,18 @@ impl<'a> Parser<'a> {
         Ok((ix, child))
     }
 
+    fn is_repeatable(&self, child: &Expr) -> bool {
+        match child {
+            Expr::LookAround(_, _) => false,
+            Expr::Empty => false,
+            Expr::StartText => false,
+            Expr::EndText => false,
+            Expr::StartLine => false,
+            Expr::EndLine => false,
+            _ => true
```

Comment:
Mind running `cargo fmt`? Looks like the build is failing because of a missing comma here.

Reply:
done :) maybe worth mentioning this in the contributing guide to potentially save time in future for new contributors? :)

Reply:
Yep done :)